bower.json

```diff
@@ -3,7 +3,8 @@
   "description": "TODO App built using MEAN",
   "main": "server.js",
   "authors": [
-    "Carlos de Dios <c.dedios@outlook.com>"
+    "Carlos de Dios <c.dedios@outlook.com>",
+    "Kenji Mukai <kensuka21@gmail.com>"
```

Comment:
Nice call, I had completely forgotten about this field we should update package.json as well.